### DISCUSSION TITLE
Help need on the root cause of  "unresolved external symbol" during compile

### DISCUSSION THREAD

**Discussion by Yuchenhenry** [ocornut/imgui#4690] [New Users Build/Link/Run issues ONLY!]
Can someone help me with this struggle? I have glfw3.lib and opengl32.lib included in my linker but the problem remains. Thank you very much. I am pretty new to ImGui and any suggestions are welcome!

![Problem_With_ImGUI](https://user-images.githubusercontent.com/90065307/139921303-276392cf-7791-4c8f-8d83-7f30fecebee3.png)

**Comment by ocornut** [comment]
You are not linking with GLEW which was the default OpenGL loader used by the GL backend, but if you update to latest version we don’t need an OpenGL loader at all anymore.

**Reply by Yuchenhenry** [reply]
Hello omar. I have include the glew32s.lib in the linker. I am using VS 2019, but the problem remains. I do also have glad used as well. Do I need to include glew32.dll in somewhere for use? 

Thank you for your reply!

**Comment by ocornut** [comment]
Update your copy of dear imgui and this won’t be a problem either way.

**Reply by Yuchenhenry** [reply]
The problem goes away after I put the newest copy in. Thank you very much omar. The Dear ImGui is really helpful for my openGL application.
